js/src/utils/helpers.js

```diff
@@ -129,12 +143,14 @@ HELPERS.playPromise = function (whichPlayer, firstPlayerPlayRequest) {
         } else if (firstPlayerPlayRequest && whichPlayer === 'content' && !this.adIsLinear) {
           if (DEBUG) {
             FW.log(e);
+            FW.log('e.message - ' + e.message);
             FW.log('initial play promise on content player has been rejected for non-linear asset - likely autoplay is being blocked');
```

Comment:
I would combine these 3, to make it cleaner. i.e ` FW.log(e)` will print both e.message and e.stack

Reply:
Agree, but the thing is that only string logs are being sent to apiv3, when there are objects in the json, the request is not being sent. Possibly because the data is too big with objects.